packages/block-library/src/gallery/gallery-image-style.native.scss

```diff
@@ -1,11 +1,12 @@
 $gallery-image-container-height: 150px;
-$overlay-border-width: 3px;
+$overlay-border-width: 2px;
 $caption-background-color: rgba(0, 0, 0, 0.4);
 $caption-padding-vertical: 8px;
```

Comment:
Can we use `$grid-unit` which is `8px` instead of creating `$caption-padding-vertical: 8px`?